akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -1588,21 +1591,19 @@ final class TakeWithin[T](val timeout: FiniteDuration) extends SimpleLinearGraph
 final class DropWithin[T](val timeout: FiniteDuration) extends SimpleLinearGraphStage[T] {
   override def createLogic(inheritedAttributes: Attributes): GraphStageLogic = new TimerGraphStageLogic(shape) with InHandler with OutHandler {
 
-    private var allow = false
+    override def preStart(): Unit = scheduleOnce("DropWithinTimer", timeout)
```

Comment:
@jrudolph Good idea. So here I am going to remove the timer. The TimerGraphStageLogic refactorization should be put in another PR.